lib/stm32/common/dma_common_f24.c

```diff
@@ -775,6 +776,20 @@ void dma_set_memory_address_1(uint32_t dma, uint8_t stream, uint32_t address)
 }
 
 /*---------------------------------------------------------------------------*/
+/** @brief DMA Stream Get the Transfer Block Size
+
+@param[in] dma unsigned int32. DMA controller base address: DMA1 or DMA2
+@param[in] stream unsigned int8. Stream number: @ref dma_st_number
+@returns unsigned int16. Number of remaining data words to transfer (65535
+maximum).
+*/
```

Comment:
I just based the description on the `_set_` call as closely as I could since I don't actually know the proper doxygen syntax ;)

So do you want me to fix the style of the rest of the stuff in the file or just keep it as is?